Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ruby:2.7
+
+# 必要なパッケージのインストール（Rails6からWebpackerがいるので、yarnをインストールする）
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
+        && echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list \
+        && apt-get update -qq \
+        && apt-get install -y build-essential libpq-dev nodejs yarn
+
+# rails用のディレクトリを作成
+RUN mkdir /myapp
```

Comment:
何か処理を書かれる際は、公式ドキュメントを一通り読むのを是非習慣にしてみてください〜。

例えばWORKDIRの項目を見ると

> If the WORKDIR doesn’t exist, it will be created even if it’s not used in any subsequent Dockerfile instruction.

とあるのでこの行の記載は不要ですね。

https://docs.docker.com/engine/reference/builder/#workdir

Reply:
公式のドキュメントをよく読んでいませんでした。 WORKDIR がなければ、作成されること理解しました。https://github.com/dokugaku-engineer/DailyReport/pull/8/commits/354ab0bfdac19aecdcc728dc4acfa9feb57f2825